src/libsyntax/ext/tt/quoted.rs

```diff
@@ -50,11 +50,17 @@ pub struct SequenceRepetition {
     /// The optional separator
     pub separator: Option<Token>,
     /// Whether the sequence can be repeated zero (*), or one or more times (+)
-    pub op: KleeneOp,
+    pub op: KleeneToken,
     /// The number of `Match`s that appear in the sequence (and subsequences)
     pub num_captures: usize,
 }
 
+#[derive(Clone, PartialEq, RustcEncodable, RustcDecodable, Debug)]
```

Comment:
Should I derive Copy? This struct should be slightly larger than Span which is Copy. Are there any style guide recommandation for when to derive Copy?

Reply:
I actually had to, because SmallVec needs the item to be Copy to extend with a slice.